network-api/networkapi/mozfest/templates/fragments/hero/large.html

```diff
@@ -3,7 +3,8 @@
 <div class="background-wrapper">
   <div class="overlay"></div>
   {% if banner_video_type == "hardcoded" %}
-    <video class="banner-video" playsinline muted loop preload poster="" >
+    <video class="banner-video" playsinline muted loop preload
+           poster="https://s3.amazonaws.com/mofo-assets/foundation/video/mozfest-banner-video-poster.png" >
```

Comment:
I put the file on s3 as welll so if we ever have to update the video, we can just update both video and the still image without a new deploy.

Reply:
nice!